ui/src/shared/utils/resourceToTemplate.ts

```diff
@@ -138,6 +150,33 @@ const cellToRelationship = (cell: Cell) => ({
   id: cell.id,
 })
 
+export const variableToTemplate = (
+  v: Variable,
+  baseTemplate = blankVariableTemplate()
+) => {
+  const variableName = _.get(v, 'name', '')
+  const templateName = `${variableName}-Template`
+  const variableData = variableToIncluded(v)
+
+  return {
+    ...baseTemplate,
+    meta: {
+      ...baseTemplate.meta,
+      name: templateName,
+      description: `template created from variable: ${variableName}`,
+    },
+    content: {
+      ...baseTemplate.content,
+      data: {
+        ...baseTemplate.content.data,
+        ...variableData,
+        relationships: {},
```

Comment:
is there a route to get labels for variables even though adding labels to variables has not been implemented yet? 
variable templates should have included labels at some point, I can't decide if we can do it now, or make an issue for it.

Reply:
@ebb-tide there isn't currently a `/variables/{id}/labels` in the swagger yet, so we'll have to wait for an `API` to update.

 Regarding template labels I also see that `baseTemplate` has `content`, `meta`, and `labels`.  Is this being used? I see most labels in `relationships` and `included`, so I wasn't sure if it could be removed.